bee-message/src/payload/transaction/essence/regular.rs

```diff
@@ -5,77 +5,62 @@ use crate::{
     constant::IOTA_SUPPLY,
     input::{Input, INPUT_COUNT_RANGE},
     output::{Output, OUTPUT_COUNT_RANGE},
-    payload::{option_payload_pack, option_payload_packed_len, option_payload_unpack, Payload},
+    payload::{OptionalPayload, Payload},
     Error,
 };
 
-use bee_common::packable::{Packable, Read, Write};
-
-use alloc::{boxed::Box, vec::Vec};
+use bee_packable::{
+    bounded::BoundedU16,
+    error::{UnpackError, UnpackErrorExt},
+    packer::Packer,
+    prefix::BoxedSlicePrefix,
+    unpacker::Unpacker,
+    Packable,
+};
 
-/// A builder to build a [`RegularTransactionEssence`].
-#[derive(Debug, Default)]
-pub struct RegularTransactionEssenceBuilder {
```

Comment:
I'm not a big fan of removing the builder tbh :(

Reply:
As long as Rust does not have named arguments, I think I prefer the builder too.

Reply:
I would agree if this had more fields, and actually removing the builder made the code shorter.

Reply:
Solved :p